libs/fills/src/lib/fills-table.tsx

```diff
@@ -0,0 +1,154 @@
+import { Side } from '@vegaprotocol/types';
+
+export interface FillsTableProps {
+  partyId: string;
+  fills: FillFields[];
+}
+
+export const FillsTable = forwardRef<AgGridReact, FillsTableProps>(
+  ({ partyId, fills }, ref) => {
+    return (
+      <AgGrid
+        ref={ref}
+        rowData={fills}
+        overlayNoRowsTemplate="No fills"
+        defaultColDef={{ flex: 1, resizable: true }}
+        style={{ width: '100%', height: '100%' }}
+        getRowId={({ data }) => data.id}
+      >
+        <AgGridColumn headerName={t('Market')} field="market.name" />
+        <AgGridColumn
+          headerName={t('Amount')}
+          field="size"
+          cellClass={({ data }: { data: FillFields }) => {
+            let className = '';
+            if (data.buyer.id === partyId) {
+              className = 'text-vega-green';
+            } else if (data.seller.id) {
+              className = 'text-vega-red';
+            }
+            return className;
+          }}
+          valueFormatter={({ value, data }: ValueFormatterParams) => {
+            let prefix;
+            if (data.buyer.id === partyId) {
+              prefix = '+';
+            } else if (data.seller.id) {
+              prefix = '-';
+            }
+
+            const size = addDecimalsFormatNumber(
+              value,
+              data.market.positionDecimalPlaces
+            );
+            return `${prefix}${size}`;
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Value')}
+          field="price"
+          valueFormatter={({ value, data }: ValueFormatterParams) => {
+            const asset =
+              data.market.tradableInstrument.instrument.product.settlementAsset
+                .symbol;
+            const valueFormatted = addDecimalsFormatNumber(
+              value,
+              data.market.decimalPlaces
+            );
+            return `${valueFormatted} ${asset}`;
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Filled value')}
+          field="price"
+          valueFormatter={({ value, data }: ValueFormatterParams) => {
+            const asset =
+              data.market.tradableInstrument.instrument.product.settlementAsset
+                .symbol;
+            const size = new BigNumber(
+              addDecimal(data.size, data.market.positionDecimalPlaces)
+            );
+            const price = new BigNumber(
+              addDecimal(value, data.market.decimalPlaces)
+            );
+
+            const total = size.times(price).toString();
+            const valueFormatted = formatNumber(
+              total,
+              data.market.decimalPlaces
+            );
+            return `${valueFormatted} ${asset}`;
+          }}
+        />
+        <AgGridColumn
+          headerName={t('Role')}
+          field="aggressor"
+          valueFormatter={({ value, data }: ValueFormatterParams) => {
+            const taker = t('Taker');
```

Comment:
Those two can be defined and translated in module scope instead of valueFormatter functon scope